config/passport.js

```diff
@@ -51,32 +52,46 @@ module.exports = function(passport, models) {
                 email: profile.emails[0].value,
                 provider: 'google'
             }, defaults: {
-                    email: profile.emails[0].value,
-                    provider: 'google',
-                    google_token: accessToken,
-                    google_id: profile.id,
-                    fname: fname,
-                    lname: lname
-                }})
+                email: profile.emails[0].value,
```

Comment:
Why is the indentation changed?

Reply:
@Raxerz  The indentation was not correct. As locally, **eslint** was giving lint warnings with it. So, I changed that to address the warnings.

Reply:
Oh okay